src/scripting/natives/natives_entities.cpp

```diff
@@ -195,6 +195,8 @@ void UnhookEntityOutput(ScriptContext& script_context)
 
 void AcceptInput(ScriptContext& script_context)
 {
+    if (!CEntityInstance_AcceptInput) return;
```

Comment:
I think we can hard-fail at this point using `script_context.ThrowNativeError()`